test/langtools/jdk/javadoc/doclet/testAnnotationTypes/TestAnnotationTypes.java

```diff
@@ -46,6 +46,7 @@ public static void main(String... args) throws Exception {
     public void test() {
         javadoc("-d", "out-1",
                 "-sourcepath", testSrc,
+                "--no-platform-link",
```

Comment:
I see lots of instances of `no-platform-link` in this and subsequent tests. `JavadocTester` does have the concept of default options, although that may be more for the behavior after executing javadoc and not for the options given to javadoc itself. Is it worth supporting default javadoc options, since that the default can be disabled for specific tests?

Reply:
I can't really find how `JavadocTester` uses or supports default options. My concern with this would be that it would make JavadocTester less transparent and intuitive to use, as you'd have to be aware what the default options are.